girder-tech-journal-gui/src/pages/view/view.js

```diff
@@ -149,6 +149,12 @@ var submissionView = View.extend({
                 },
                 logo: logoURL
             }));
+
+            // Import JS and CSS needed for share buttons after render
+            // ******************************************************
+            require('social-likes-next');
```

Comment:
@josephsnyder It may cause problems to use `require` like this. I can talk more next week, if you'd like.

Reply:
Thanks Brian,  @waxlamp and I thought this was one way to get it to work, but not the ideal solution. I'd be happy to talk about it next week to find the proper way to do it

Reply:
After talking, we found the [Manual Initialization](https://github.com/sapegin/social-likes-next#manual-initialization)  steps for social-likes where we can use "import" but only call it it when the DOM object exists